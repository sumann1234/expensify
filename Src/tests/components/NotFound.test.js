import React from 'react';
 import {shallow} from 'enzyme';
import DoesNotExist from '../../components/NotFoundPage';


test('testing the NotFoundPage',() =>{
    const wrapper= shallow(<DoesNotExist/>);
    expect((wrapper)).toMatchSnapshot();
})