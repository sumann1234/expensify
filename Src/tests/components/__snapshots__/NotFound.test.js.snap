// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testing the NotFoundPage 1`] = `
<h1>
  ERROR 404: NOT FOUND!
  <[object Object]
    to="/"
  >
     Go home
  </[object Object]>
</h1>
`;
